Gem/Code/Source/UserSettings/MultiplayerSampleUserSettings.cpp

```diff
@@ -180,6 +188,93 @@ namespace MultiplayerSample
         }
     }
 
+    bool MultiplayerSampleUserSettings::GetFullscreen()
+    {
+        bool fullscreen = false;
+
+        if (auto* registry = AZ::SettingsRegistry::Get(); registry != nullptr)
+        {
+            registry->Get(fullscreen, m_fullscreenKey.c_str());
+        }
+
+        return fullscreen;
+    }
+
+    void MultiplayerSampleUserSettings::SetFullscreen(bool fullscreen)
+    {
+        if (auto* registry = AZ::SettingsRegistry::Get(); registry != nullptr)
+        {
+            if (AZ::IConsole* console = AZ::Interface<AZ::IConsole>::Get(); console)
+            {
+                // Change the fullscreen state if we haven't created the window yet.
+                AZ::CVarFixedString commandString = AZ::CVarFixedString::format("r_fullscreen %u", fullscreen ? 1 : 0);
+                console->PerformCommand(commandString.c_str());
+
+                // Change the fullscreen state if the window already exists
+                AzFramework::NativeWindowHandle windowHandle = nullptr;
+                AzFramework::WindowSystemRequestBus::BroadcastResult(
+                    windowHandle,
+                    &AzFramework::WindowSystemRequestBus::Events::GetDefaultWindowHandle);
+
+                AzFramework::WindowRequestBus::Event(
+                    windowHandle,
+                    &AzFramework::WindowRequestBus::Events::SetFullScreenState, fullscreen);
+            }
+
+            registry->Set(m_fullscreenKey.c_str(), fullscreen);
+        }
+    }
+
+    AZStd::pair<uint32_t, uint32_t> MultiplayerSampleUserSettings::GetResolution()
+    {
+        uint64_t width = 1920;
+        uint64_t height = 1080;
+
+        if (auto* registry = AZ::SettingsRegistry::Get(); registry != nullptr)
+        {
+            registry->Get(width, m_resolutionWidthKey.c_str());
+            registry->Get(height, m_resolutionHeightKey.c_str());
+        }
+
+        return { aznumeric_cast<uint32_t>(width), aznumeric_cast<uint32_t>(height) };
+    }
+
+    void MultiplayerSampleUserSettings::SetResolution(AZStd::pair<uint32_t, uint32_t> resolution)
+    {
+        if (auto* registry = AZ::SettingsRegistry::Get(); registry != nullptr)
+        {
+            if (AZ::IConsole* console = AZ::Interface<AZ::IConsole>::Get(); console)
+            {
+                // This is roughly what it takes to change the screen resolution, but it gets complicated by Dpi scaling
+                // and whether or not the window is currently fullscreen. The code below is temporarily removed until the
+                // logic can be enhanced to account for these other conditions.
+
+                /*
+                // Change the resolution if the window doesn't exist yet.
```

Comment:
Dead code?

Reply:
Yes, but intentionally left in dead code as an example of how to do it if you don't care about perfection. I can accept it if you want me to remove it, I would spiral through and remove all of the functions and toggle code related to supporting resolution then.

Reply:
Maybe a comment pointing folks in the right direction might be better then a chunk of commented out code in that case..
```
// If the window is not yet created, you can set the r_width and r_height console variables to alter the window dimensions
// If the window is created, use the AzFramework::NativeWindowHandle to retrieve the active window handler and then use the AzFramework::WindowRequestBus to alter the dimensions of the active window
```
And also a quick comment as to why you ultimately ended up going down a different path.

Reply:
Ahh, I missed the 'comment why you went a different route' because I was distracted by the dead code 🤣

Reply:
lol. It's not a different route or anything yet, it's just that I can't make it work in a user-friendly way without extending our window APIs in AzFramework. I'll continue making this work in development, I'm just limited in how far I can take it in stabilization.

However, given your concern about the dead code, I'm seeing if I can make the code active but not behave differently than now by default.

Reply:
Oh! I think I understand you better now, resolution setting changes from the settings screen are actually unsupported right now with that code removed. With the registry code it actually requires a program restart.. I didn't clue in on that at all. That removes my concerns about dead code, this is actually essential to the correct functioning of the feature.

Reply:
I'm okay with you leaving this commented out here and continuing to iterate on this in dev.

Reply:
Ok, it's now active code. You can technically change the resolution in your usersettings.setreg file, but it's not exposed in the UI.